### PR TITLE
Gssoc 24v wanna add my project via #971

### LINKED ISSUE #971: GSSOC'24 wanna add my project virtual classes attendance extractor
## ❓ Questions and Help
the project description is that if we upload the attendance sheet which include the time duration in the virtual class , students list with rolls, and the minimum threshold percentage of class duration attended. the sorted list of students  will be displays. technologies used python libs like flask for backend, pandas for data analysis, etc

can i add this to your repo . i think it can be a intermediate one

### PR DESCRIPTION
**Pull request by ramsatyasai**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?
yes
(Write your answer here.)

### Description
the project description is that if we upload the attendance sheet which include the time duration in the virtual class , students list with rolls, and the minimum threshold percentage of class duration attended. the sorted list of students will be displays. technologies used python libs like flask for backend, pandas for data analysis, etc
(Write your answer here.)

Fixes #971 

### Checklist

- [. ] I've read the contribution guidelines.
- [. ] I've checked the issue list before deciding what to submit.
- [. ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
please add Gssoc tags to the issue before merging the pull request 
(Write your answer here.)

### CODE REVIEW

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@ramsatyasai Add a good `readme.md` file where you explain about your project and makesure add the deploy link of the project.

**Comment by ramsatyasai** [comment]
done the changes required can you please verify and merge the pull request

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@ramsatyasai Yet it not full fill the requirement , as i mentioned you need to host your website and add the deploy link here in `readme.md` file.

**Comment by ramsatyasai** [comment]
once check it sir just deployed the application

File: Web Development/Basic/virtual-classes-attendance-automation-system/readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+The Virtual Classes Attendance Automation System is a web-based application developed using Flask, a Python web framework, to automate the process of generating attendance reports for virtual classes. This system allows users to upload two CSV files containing student information and attendance records, specify a minimum duration threshold for attendance, and generate a detailed report based on the provided criteria.
+## Deployment
+
+Check out the live deployment of this project [here](https://virtual-classes-attendance-automation.onrender.com).
+
```
Comment (Kushal997-das) [inline Web Development/Basic/virtual-classes-attendance-automation-system/readme.md:5]: The site is not loading , better to deploy platform like `netlify` or `github-pages`.

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@ramsatyasai Kindly fix all these then say you complete not before .

File: Web Development/Basic/virtual-classes-attendance-automation-system/readme.md (hunk 1 above)
Comment (ramsatyasai) [reply Web Development/Basic/virtual-classes-attendance-automation-system/readme.md:5]: https://github.com/Kushal997-das/Project-Guidance/assets/108385414/8522a05d-6984-4326-9cde-fb81351d47f4

Before updating you the changes I have checked it twice. Maybe some network issue on ur side. The link is working fine even I have confirmed it with my friends.

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
Okey! Now add your project name [here](https://github.com/Kushal997-das/Project-Guidance/blob/main/Web%20Development/README.md) alpha format .

I'll check it later.

**Comment by ramsatyasai** [comment]
done

**Review by Kushal997-das — APPROVED** [review]
@ramsatyasai 💯

**Pull request merged**